src/plugins/intel_gna/src/ops/util/util.hpp

```diff
@@ -142,6 +142,7 @@ inline bool is_gna_non_functional_node(const std::shared_ptr<ngraph::Node>& node
     return std::dynamic_pointer_cast<ngraph::opset8::Reshape>(node) ||
             std::dynamic_pointer_cast<ngraph::opset8::Squeeze>(node) ||
             std::dynamic_pointer_cast<ngraph::opset8::Unsqueeze>(node) ||
+            std::dynamic_pointer_cast<ngraph::opset8::FakeQuantize>(node) ||
```

Comment:
It should not be here, in common part. FQ is included also in list of activations. It it better handle it in transformation itself.

Reply:
ok, removed